shopfloor_mobile/static/wms/src/wms_utils.js

```diff
@@ -237,14 +239,15 @@ export class WMSUtils {
         });
         return options;
     }
-    move_line_qty_picker_options(line, override = {}) {
```

Comment:
I would not remove this. I would just deprecate it.

Reply:
done